docs/how-to-use-playground.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Description } from '@storybook/addon-docs';
+import playgroundLandingPage from './images/playground-landing-page.png'
+
+<Meta
+  title="Documentation/How To Use Playground"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+# How to use playground?
+Playground is a demonstration area for developers. It stays under the `playground` folder in the root. It helps you to test your features & components by eliminating the need for stories.
+
+<bl-alert variant="warning" caption="Note">You should not commit the changes in the playground folder.</bl-alert>
+
+You can run it with `npm run serve`, then it builds project and run the build in `watch mode` at port `8000`. You can either create a `foo.html` file and work on it or simply use `template.html`.
```

Comment:
I think we should suggest starting with copying `template.html` file as `foo.html` and work on it. As `template` name comes from this intention. In this case user will not need to extra attention to not commit changes accidentally, since `playground` folder is already ignored from git, except `template.html` file.